examples/graph/compute_spatial_neighbors.py

```diff
@@ -47,13 +46,7 @@
 # We can visualize the neighbors of a point to better visualize what `n_rings` mean:
 _, idx = adata.obsp["spatial_connectivities"][420, :].nonzero()
 idx = np.append(idx, 420)
-sc.pl.spatial(
-    adata[idx, :],
-    neighbors_key="spatial_neighbors",
-    edges=True,
-    edges_width=1,
-    img_key=None,
-)
+sq.pl.spatial_scatter(adata[idx, :], shape=None, connectivity_key="spatial_connectivities")
```

Comment:
don't use shape = None otehrwise size is not correct and can't see node. This is how it renders now:
![image](https://user-images.githubusercontent.com/25887487/174770727-6ecb9bd1-bfdd-479e-8e0a-07d761d198c6.png)

Reply:
Fixed